Kernel: Print panic backtrace to both the screen and serial

Lubrsi opened the pull request:
Previously it would only print the backtrace to serial, which would be
inaccessible if you don't have serial setup.

bgianfo commented on Kernel/KSyms.h:
```diff
@@ -23,6 +23,6 @@ extern bool g_kernel_symbols_available;
 extern FlatPtr g_highest_kernel_symbol_address;
 
-void dump_backtrace();
+void dump_backtrace(bool critical = false);
```
Should we use an enum instead?

awesomekling replied:
Yeah that seems like a good idea.

Lubrsi commented:
Addressed the comment :+1:. Here's an example:
![Screenshot from 2021-08-04 16-51-39](https://user-images.githubusercontent.com/25595356/128213294-f46189d4-3857-4d9d-afdf-5b089737dc1c.png)
